test/jdk/java/awt/print/PrinterJob/DrawStringMethods.java

```diff
@@ -21,44 +21,60 @@
  * questions.
  */
 
-/**
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.font.FontRenderContext;
+import java.awt.print.Book;
+import java.awt.print.PageFormat;
+import java.awt.print.Printable;
+import java.awt.print.PrinterJob;
+import java.text.AttributedCharacterIterator;
+import java.text.AttributedString;
+
+import jtreg.SkippedException;
+
+/*
  * @test
  * @bug 4185019
  * @key printer
  * @summary Confirm that all of the drawString methods on Graphics2D
- *          work for printer graphics objects.
+ * work for printer graphics objects.
```

Comment:
I suggest keeping the original style, I think it is easier to parse the summary when the wrapped line is indented with spaces.